devel/turbogit/Portfile

```diff
@@ -13,12 +13,12 @@ maintainers         {b4nst @b4nst} openmaintainer
 description         Your git boat friend
 long_description    Cli tool built to help you deal with your day-to-day git work.
 
-github.tarball_from releases
-worksrcdir          .
+github.tarball_from archive
+worksrcdir          turbogit-${version}
```

Comment:
is setting `worksrcdir` required, I thought that the default values from the github PG should do the trick. Please see if you can remove this.

Reply:
Nice catch, thanks!